src/ProjectModule/Components/Home/Home.tsx

```diff
@@ -1,7 +1,177 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import styles from "./Home.module.css"
+import { Doughnut } from 'react-chartjs-2';
+import axios from 'axios';
+import {Tooltip,Title,ArcElement,Legend,Chart as ChartJS} from "chart.js"
+import { baseUrl } from '../../../Constants/Components/Urls';
+ChartJS.register(Tooltip,Title,ArcElement,Legend);
+
 
 export default function Home() {
+const[taskToDo,setTaskToDo]=useState(0);
+const[taskProgres,setTaskProgres]=useState(0);
+const[taskDone,setTaskDone]=useState(0);
+const[userActive,setUserActive]=useState(0);
+const[userInActive,setUserInActive]=useState(0);
+
+
+
+const countTasks=async()=>{
+  try {
+    const token=localStorage.getItem("userToken")
+    const response=await axios.get(`${baseUrl}/Task/count`,{headers:{Authorization:token}}) 
+    setTaskToDo(response.data.toDo);
+    setTaskProgres(response.data.inProgress);
+    setTaskDone(response.data.done);
+  
+    console.log(response?.data);
+   
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+
+const activeTasks=async()=>{
+  try {
+    const token=localStorage.getItem("userToken")
+    const response=await axios.get(`${baseUrl}/Users/count`,{headers:{Authorization:token}}) 
+    setUserActive(response.data.activatedEmployeeCount);
+    setUserInActive(response.data.deactivatedEmployeeCount);
+   
+    console.log(response?.data);
+  } catch (error) {
+    console.log(error)
+  }
+}
+const dataProgress = {
+  datasets: [{
+      data: [taskToDo,taskProgres,taskDone],
+      backgroundColor:[
+        " rgb(198, 160, 182)",
+       'rgb(67, 135, 64)',
+       " rgb(168, 176, 249)"
+      ],
+  }],
+
+  // These labels appear in the legend and in the tooltips when hovering different arcs
+  labels: [
+      'ToDo',
+      "inProgress",
+      "Done"
+  ]
+};
+
+const dataActive = {
+  datasets: [{
+      data: [userActive,userInActive],
+      backgroundColor:[
+        " rgb(114, 154, 145)",
+       ' rgb(217, 138, 147)',
+      
+      ],
+  }],
+
+  // These labels appear in the legend and in the tooltips when hovering different arcs
+  labels: [
+      "Active",
+      "InActive"
+  ]
+};
+
+useEffect(()=>{
+  countTasks();
+  activeTasks();
```

Comment:
Only managers are allowed to get the users count, we need to call this function according to the current userRole.